MatchThreeBoardTest.java

```diff
@@ -120,8 +121,74 @@ public void nullTokens2Test()
         exception.expect(IllegalArgumentException.class);
         new MatchThreeBoard(null, "  ;  ");
     }
+    
+    @Test
+    public void nullPositionTest1()
```

Comment:
Please put the number of the test before the word `Test` to be consistent with the other tests.